falco/templates/_helpers.tpl

```diff
@@ -229,3 +236,68 @@ be temporary and will stay here until we move this logic to the falcoctl tool.
     - mountPath: /gvisor-config
       name: falco-gvisor-config
 {{- end -}}
+
+
+{{- define "falcoctl.initContainer" -}}
+- name: falcoctl-artifact-install
+  image: {{ include "falcoctl.image" . }}
+  imagePullPolicy: {{ .Values.falcoctl.image.pullPolicy }}
+  args: 
+    - artifact
+    - install
+  {{- with .Values.falcoctl.artifact.install.args }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.falcoctl.artifact.install.resources }}
+  resources:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  securityContext:
+  {{- if .Values.falcoctl.artifact.install.securityContext }}
+    {{- toYaml .Values.falcoctl.artifact.install.securityContext | nindent 4 }}
+  {{- end }}
+  volumeMounts:
+    - mountPath: /plugins
+      name: plugins-install-dir
+    - mountPath: /rulesfiles
+      name: rulesfiles-install-dir
```

Comment:
Probably, `/plugins` and `/rulesfiles` shouldn't be hardcoded.
See my comment below (the one attached to the config).